.travis.yml

```diff
@@ -27,7 +27,7 @@ addons:
     packages:
       - python2.4
 install:
-  - pip install tox
+  - pip install tox yamllint
```

Comment:
Since yamllint is only being used by shippable and not travis, there's no need to update .travis.yml.